src/microcosm.js

```diff
@@ -39,6 +39,18 @@ export default class Microcosm extends Emitter {
 
     // Standard store reduction behaviors
     this.addStore(MetaStore)
+
+    // Microcosm is now ready. Call the initialize method
```

Comment:
Oops. Duh. I called this initialize before, but then changed it to `setup` because JS class libraries have a history of using `init` and `initialize` sort of like a constructor.